package/score-worker/src/main/resources/META-INF/spring/score/context/scoreWorkerSchedulerContext.xml

```diff
@@ -15,6 +15,8 @@
         <task:scheduled ref="sessionDataHandler" method="sessionTimeOutScheduler" fixed-rate="60000"/>
         <task:scheduled ref="workerExecutionMonitorService" method="executeScheduledWorkerMonitors" fixed-delay="#{scheduledWorkerMonitorInterval}"/>
         <task:scheduled ref="workerExecutionMonitorService" method="collectMonitorInformation" fixed-delay="#{workerMonitorRefreshInterval}"/>
+        <task:scheduled ref="workerMetricCollectorService" method="collectPerfMetrics" fixed-delay="#{scheduledPerfMetricCollectionInterval}"/>
+        <task:scheduled ref="workerMetricCollectorService" method="collectPerfMetricsInBatches" fixed-delay="#{scheduledQueueMetricCollectionInterval}"/>
```

Comment:
scheduledQueueMetricCollectionInterval should be named as scheduledMetricDispatchInterval